archivist/storage_integrity.py

```diff
@@ -0,0 +1,16 @@
+"""Archivist Storage integrity
+
+   Only 2 values allowed that correspond to the backend definitions.
```

Comment:
Is this ‘2 values’ a sensible header/documentation ? It doesn’t really help anyone and it’ll get stale quickly

Reply:
redone